straxen/plugins/merged_s2s/merged_s2s.py

```diff
@@ -17,7 +17,7 @@ class MergedS2s(strax.OverlapWindowPlugin):
     """
     __version__ = '1.0.2'
 
-    depends_on = ('peaklets', 'peaklet_classification', 'lone_hits')
+    depends_on = ('peaklets', 'peaklet_classification_som', 'lone_hits')
```

Comment:
I thought we said your plugin would be a drop-in replacement for the peaklet classification. So why are you adding it as an extra data_type?